src/progress/progress_set.rs

```diff
@@ -155,7 +155,7 @@ pub struct ProgressSet {
     #[get = "pub"]
     next_configuration: Option<Configuration>,
     configuration_capacity: (usize, usize),
-    // A preallocated buffer for sorting in the minimally_commited_index function.
+    // A preallocated buffer for sorting in the minimally_committed_index function.
```

Comment:
Did you rename the actual function as well?

Reply:
I can't find this function, just found ***maximal_committed_index***

Reply:
Then I'll state this as a bigger problem -- it's probably a non-trivial typo.

Git blame shows that this commit is https://github.com/pingcap/raft-rs/tree/88e3697c985eb55c599add7b5cd24a40fd05ebc3, where at that time I can't find such function as well :(

/cc @Hoverbear

Reply:
<pre><font color="#3465A4">~/git-repos/raft-rs</font><font color="#2E3436">|(88e3697...)</font> <font color="#C4A000">λ&gt;</font> grep -rnE &quot;minimally_commited_index&quot; src
<font color="#75507B">src/progress.rs</font><font color="#06989A">:</font><font color="#4E9A06">82</font><font color="#06989A">:</font>    // A preallocated buffer for sorting in the <font color="#EF2929"><b>minimally_commited_index</b></font> function.
</pre>

Reply:
It's a stale comment. Just rename it to `maximal_committed_index` is OK.

Reply:
Yup that's my bad, sorry.